code/datums/gamemodes/game_mode.dm

```diff
@@ -420,6 +420,8 @@ Sensors indicate [numXenosShip || "no"] unknown lifeform signature[numXenosShip
 		dat += "[GLOB.round_statistics.grenades_thrown] total grenades exploding."
 	else
 		dat += "No grenades exploded."
+	if(GLOB.round_statistics.total_human_deaths)
+		dat += "[GLOB.round_statistics.total_human_deaths] people were killed, among which [GLOB.round_statistics.total_human_revives] were revived. For a [(GLOB.round_statistics.total_human_revives / max(GLOB.round_statistics.total_human_deaths, 1)) * 100]% medical revival rate."
```

Comment:
missing human revives

Reply:
I added the var to round_statistics and it increments on defib revive. What am I not seeing?